recipes/agc/meta.yaml

```diff
@@ -1,20 +1,26 @@
-{% set version = "3.0" %}
-{% set system = "linux"%}  # [linux]
-{% set system = "mac"%}  # [osx]
-{% set sha256 = "3ee3245e556221d246f9afe5d9ad86888f4eb5aea6b10f621241c3170ef8835e" %}  # [linux]
-{% set sha256 = "3d68b6b76948b6d9a36c4e848e37969eda6ae0ec1abace4ca61a7cc585dcdabf" %}  # [osx]
+{% set version = "3.1" %}
+
 package:
   name: agc
   version: {{ version }}
+
 build:
   number: 0
   script: "mkdir -p ${PREFIX}/bin; cp agc ${PREFIX}/bin"
+  run_exports:
+    - {{ pin_subpackage('agc', max_pin="x") }}
 source:
-  url: https://github.com/refresh-bio/agc/releases/download/v{{ version }}/agc-{{ version }}_x64-{{ system }}.tar.gz
-  sha256: {{ sha256 }}
+  - url: https://github.com/refresh-bio/agc/releases/download/v{{ version }}/agc-{{ version }}_arm_linux.tar.gz # [aarch64]
+    sha256: 5fee3ccbb2691d82a877987e204c5946dbdbe40b9603af5d2421cb8506577534                                           # [aarch64]
+  - url: https://github.com/refresh-bio/agc/releases/download/v{{ version }}/agc-{{ version }}_x64_linux.tar.gz   # [linux64]
+    sha256: 4868dd45c1ed31a7f36bcbc536174249de27b87ec83858d22b128cbec8deef53                                           # [linux64]
+  - url: https://github.com/refresh-bio/agc/releases/download/v{{ version }}/agc-{{ version }}_x64_mac.tar.gz       # [osx and x86_64]
+    sha256: 0ef911e6bb8df791593ee6e89b5bddbf82b5823684955d249d6a67a39e6d0d4e                                           # [osx and x86_64]
+  - url: https://github.com/refresh-bio/agc/releases/download/v{{ version }}/agc-{{ version }}_m1_mac.tar.gz       # [osx and arm64]
+    sha256: ff8cc5dd4c5825a41e042567a93e9e01ef43fc91624c983640a29fd3c7ac03e4                                           # [osx and arm64]
```

Comment:
Can this be compiled per platform rather than downloading binaries? As-is, could have issues with ABI incompatibility when installed into an environment with other packages.

Reply:
The OSX compilation requires the GNU Compiler (g++-11) instead of Clang, which is used by bioconda. The "-static-libgcc"/"-static" flags were used when linking (osx/linux), I thought it would be sufficient.

Reply:
sounds reasonable; fingers crossed!